### PR TITLE
[FIX] have explicit modules on a branch override repo modules, not add to them

### PR DESCRIPTION
**Pull request by hbrunn**
no idea why I though they should be merged before

### CODE REVIEW

File: runbot_buildout/models/runbot_build.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def _job_10_test_base(self, build, lock_path, log_path):
                     if '/%s/' % parts_path in dirname:
                         repo_modules.append(basename)
             explicit_modules = (
-                (build.branch_id.modules or '') + ',' +
+                (build.branch_id.modules or '') or
```
Comment (moylop260) [inline runbot_buildout/models/runbot_build.py]: What about `build.branch_id.modules or build.repo_id.modules or ''`?

**Review by moylop260 — APPROVED** [review]
small nitpick but not blocking

👍

File: runbot_buildout/models/runbot_build.py (hunk 1 above)
Comment (hbrunn) [reply runbot_buildout/models/runbot_build.py]: much better, thanks

**Comment by moylop260** [comment]
Red CI is unrelated for module changed here.
Merging since that it is small fix and there are not too much reviewers for this module.

**Pull request merged**